Add postmortem feature updates (#36690)
DataDog/documentation: content/en/incident_response/incident_management/post_incident/follow-ups.md

**From Slack**: In the incident channel, run `/datadog followup` to create a new follow-up or `/datadog followup list` to view and manage existing follow-ups. For more Slack commands, see [Integrate Slack with Datadog Incident Management][5].

## Follow-ups in postmortem notebooks

> You can display follow-ups directly in a postmortem notebook using the `{{incident.follow-ups}}` template variable. When added to a Datadog Notebooks postmortem template, this variable renders a list of follow-ups that you can assign, set due dates for, and create new ones from within the notebook. For more information, see [Incident Postmortems][6].

Reviewer: ```suggestion
You can display follow-ups directly in a postmortem notebook using the `{{incident.follow-ups}}` template variable. When added to a Datadog Notebooks postmortem template, this variable renders a list of follow-up items. From the list view in your notebook, you can set due dates, assign items, or create new follow-up items. For more information, see [Incident Postmortems][6].
```
